static/templates/favicon.svg.hbs

```diff
@@ -0,0 +1,22 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      d="M15.1773 2.39308C15.1773 3.1991 14.8153 3.91526 14.2636 4.34953L8.90867 9.13165C8.80932 9.21698 8.68001 9.0796 8.75156 8.96455L10.7158 5.0317C10.7707 4.92165 10.6994 4.78617 10.5865 4.78617H2.96804C1.78814 4.78617 0.822632 3.70918 0.822632 2.39308C0.822632 1.07698 1.78814 0 2.96804 0H13.0317C14.2118 0 15.1773 1.07698 15.1773 2.39308V2.39308ZM2.96804 16H13.0317C14.2116 16 15.1771 14.923 15.1771 13.6069C15.1771 12.2908 14.2116 11.2138 13.0317 11.2138H5.41317C5.30027 11.2138 5.22873 11.0783 5.28387 10.9683L7.24816 7.03545C7.3197 6.92064 7.19063 6.78302 7.09104 6.86835L1.73632 11.6505C1.18464 12.0847 0.822632 12.8009 0.822632 13.6069C0.822632 14.923 1.78814 16 2.96804 16V16Z"
+      fill="url(#paint0_linear)" />
+    <defs>
+        <linearGradient id="paint0_linear" x1="0.822632" y1="0" x2="0.822632" y2="16" gradientUnits="userSpaceOnUse">
+            <stop stop-color="#50ADFF"></stop>
+            <stop offset="1" stop-color="#7877FC"></stop>
+        </linearGradient>
+    </defs>
+    {{#if count}}
+        {{#if (eq count "infinite")}}
+        <path
+          d="M12.5692 10C11.2266 10 10.1764 10.7744 9.5 11.4738C8.82359 10.7744 7.77344 10 6.43078 10C4.53969 10 3 11.3463 3 13C3 14.6538 4.53969 16 6.43078 16C7.77344 16 8.82359 15.2256 9.5 14.5263C10.1764 15.2256 11.2266 16 12.5692 16C14.4603 16 16 14.6538 16 13C16 11.3463 14.4603 10 12.5692 10ZM6.43078 14.2C5.61422 14.2 4.95 13.6619 4.95 13C4.95 12.3381 5.61422 11.8 6.43078 11.8C7.20672 11.8 7.92172 12.4769 8.34016 13C7.92578 13.5175 7.20469 14.2 6.43078 14.2V14.2ZM12.5692 14.2C11.7933 14.2 11.0783 13.5231 10.6598 13C11.0742 12.4825 11.7953 11.8 12.5692 11.8C13.3858 11.8 14.05 12.3381 14.05 13C14.05 13.6619 13.3858 14.2 12.5692 14.2Z"
+          fill="white" />
+        {{else}}
+        <text x="4" y="15" fill="black" font-size="12px" font-family="source sans" stroke="white" stroke-width="1">
```

Comment:
Your screenshot shows a font that’s clearly not Source Sans. We probably want Source Sans since it’d be more readable at small sizes than the browser’s default serif fallback font, but we need to include the CSS to load it.